app/src/main/res/values/strings.xml

```diff
@@ -573,5 +573,5 @@
 
     <string name="whatsnew_title">What\'s new</string>
     <string name="whatsnew_item1_title">Meet the new Safe!</string>
-    <string name="whatsnew_item1_description">Gnosis Safe rebranded to Safe following a successful spin-off from the Gnosis DAO.\n\nNow and over the coming versions, you will see a new look and a better user experience for your ever secure Gnosis Safe.</string>
+    <string name="whatsnew_item1_description">&lt;b&gt;Gnosis Safe &lt;/b&gt; rebranded to &lt;b&gt;Safe&lt;/b&gt;. Following a successful spin-off vote from Gnosis in GIP-29, we are rebranding to Safe.\n\nNow and over the coming time, you will see a new look and a better visual experience to your ever secure Gnosis Safe that you love.</string>
```

Comment:
I would rather work with Spannables and set ForegroundColorSpan for highlighted elements

Reply:
also highlighted elements are not bold, but have different color in design mocks